net/instaweb/rewriter/css_inline_filter.cc

```diff
@@ -111,11 +112,21 @@ void CssInlineFilter::StartDocumentImpl() {
 
 CssInlineFilter::~CssInlineFilter() {}
 
+void CssInlineFilter::StartElementImpl(HtmlElement* element) {
+
+  // w3c incompatible when css inlining in body with pedantic enabled
+  if (element->keyword() == HtmlName::kBody &&
+      driver()->options()->Enabled(RewriteOptions::kPedantic)) {
+    inlining_not_pedantically_spec_ = true;
```

Comment:
I don't understand what's being tracked here.  This function gets called for every document element when inlining is on, and there will always be a 'body' element.  I think this is tracking whether we are inside the 'body' in which case it might be less confusing to call this variable in_body_ (contradicting my suggestion above).